Makefile

```diff
@@ -1,5 +1,5 @@
 #TODO add default features here
-export FEATURES?=sctp performance
+export FEATURES?=sctp performance dpdk
```

Comment:
do we expect dpdk tests to work on the gcp public cloud infrastructure, or should it be limited to baremetal? It's unclear what the network requirements are here

Reply:
Not only that, those features are those installed (& instrumented) by default on gcp CI.

For this reason, dpdk should not be there, the same way sriov or ptp are not there.